maintainers/maintainer-list.nix

```diff
@@ -5484,6 +5484,12 @@
     githubId = 346094;
     name = "Michael Alyn Miller";
   };
+  manojkarthick = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add manojkarthick``.